core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1744,6 +1744,28 @@
   @InterfaceStability.Unstable
   public static final String CARBON_SEARCH_MODE_SCAN_THREAD = "carbon.search.scan.thread";
 
+  /**
+   * The size of thread pool used for reading files in Work for CarbonReader. By default,
+   * it is number of cores in Worker
+   */
+  @CarbonProperty
+  @InterfaceStability.Unstable
+  public static final String CARBON_READER_THREAD = "carbon.reader.thread";
+
+  /**
+   * Whether it support CarbonReader, default value is false
+   */
+  @CarbonProperty
+  @InterfaceStability.Unstable
+  public static final String CARBON_READER_SUPPORT = "carbon.reader.support";
```

Comment:
Does this mean enable SDK reader ? If yes, please change property name to enable.sdk.query.executor.

Reply:
yeah, ok